routesrv/polling.go

```diff
@@ -90,6 +72,9 @@ func (p *poller) poll(wg *sync.WaitGroup) {
 		case routesCount == 0:
 			log.Error(LogRoutesEmpty)
 
+			p.metrics.UpdateGauge("routes_count", 0)
+			p.metrics.UpdateGauge("routes_byte", 0)
```

Comment:
I think we should not set metrics to zero here as we do not reset routes.

Reply:
Maybe `p.metrics.IncCounter("routes_empty")`

Reply:
Yes good idea!